grade-management/Ahk.GradeManagement.Data/Internal/GradeRepository.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using Ahk.GradeManagement.Data.Entities;
+
+namespace Ahk.GradeManagement.Data.Internal
+{
+    public class GradeRepository : IGradeRepository
+    {
+        public AhkDbContext Context { get; set; }
+        public GradeRepository()
+        {
+        }
+
+        public async Task AddGrade(Grade value)
```

Comment:
itt is látszik, hogy nincs sok értelme a repository mintának, mert lényegében csak az EF-es azonos múveleteket wrappeljük be